docs/uploader/aws-s3-multipart.mdx

```diff
@@ -386,9 +386,9 @@ The default implementation calls out to Companion’s S3 signing endpoints.
 Pass an array of field names to limit the metadata fields that will be added to
 upload as query parameters.
 
+- Set this to an empty array `false` to not send any fields.
 - Set this to `['name']` to only send the `name` field.
-- Set this to `null` (the default) to send _all_ metadata fields.
-- Set this to an empty array `[]` to not send any fields.
+- Set this to `true` (the default) to send _all_ metadata fields.
```

Comment:
shouldn't we also document what happens if you set it to an empty array? does it mean send **all** fields or **no** fields?